<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BatchCacheUi</class>
 <widget class="QWidget" name="BatchCacheUi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>597</width>
    <height>471</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Batch cache</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_7">
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>80</height>
        </size>
       </property>
       <property name="text">
        <string>Mode</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="mode">
       <property name="minimumSize">
        <size>
         <width>77</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>77</width>
         <height>16777215</height>
        </size>
       </property>
       <item>
        <property name="text">
         <string>Shotgun</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Disk</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="mode_info">
       <property name="text">
        <string>The list of assets has been found by opening the latest version of each scene.</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>4</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>20</height>
            </size>
           </property>
           <property name="text">
            <string>Shots</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="HPushButton" name="refresh">
           <property name="maximumSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>P:/global/code/pipeline/bootstrap/psycons/icon_packs/EMOJI/icon.2347.png</normaloff>P:/global/code/pipeline/bootstrap/psycons/icon_packs/EMOJI/icon.2347.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="HListWidget" name="shots">
         <property name="selectionMode">
          <enum>QAbstractItemView::ExtendedSelection</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>20</height>
          </size>
         </property>
         <property name="text">
          <string>Steps</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="HListWidget" name="steps">
         <property name="selectionMode">
          <enum>QAbstractItemView::ExtendedSelection</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>20</height>
          </size>
         </property>
         <property name="text">
          <string>Tasks</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="HListWidget" name="tasks">
         <property name="selectionMode">
          <enum>QAbstractItemView::ExtendedSelection</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>20</height>
          </size>
         </property>
         <property name="text">
          <string>Assets</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="HListWidget" name="assets">
         <property name="selectionMode">
          <enum>QAbstractItemView::ExtendedSelection</enum>
         </property>
         <item>
          <property name="text">
           <string>apple</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>pear</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>banana</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="info">
         <property name="text">
          <string>Matched:
10 caches
5 work files</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <widget class="QCheckBox" name="hide_omitted">
           <property name="text">
            <string>Hide omitted</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="stale_only">
           <property name="text">
            <string>Out of date only</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="farm">
           <property name="text">
            <string>Use farm</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="cache">
         <property name="text">
          <string>Cache</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Cache the selected work files by opening them, updating the assets to the latested version and then caching</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>HListWidget</class>
   <extends>QListWidget</extends>
   <header location="global">hlistwidget.h</header>
   <slots>
    <slot>blah()</slot>
    <slot>redraw()</slot>
   </slots>
  </customwidget>
  <customwidget>
   <class>HPushButton</class>
   <extends>QPushButton</extends>
   <header>hpushbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
